packages/cli/generators/project/templates/README.md.ejs

```diff
@@ -1,3 +1,30 @@
 # <%= project.name %>
 
-[![LoopBack](https://github.com/strongloop/loopback-next/raw/master/docs/site/imgs/branding/Powered-by-LoopBack-Badge-(blue)-@2x.png)](http://loopback.io/)
+<% if (project.description) { -%>
+> <%= project.description %>
+<% } -%>
+
+This project was generated with [LoopBack CLI](https://github.com/strongloop/loopback-next/tree/master/packages/cli) version <%= cliVersion %>.
+
+## First steps
+
+### Installation
+
+To proceed with the installation of the dependencies in Loopback you must have `nodejs` >= 10 and `npm` installed
+```sh
+npm install
+```
+> The dependencies are already installed for LoopBack 4 application scaffolded using `lb4` command. If you add other dependencies, please make sure you run `npm i` before starting the application.
+### Run
+To start the project is as easy as this
+```sh
+npm start
+```
+
```

Comment:
So with `### Run`  as the first step in `## First steps`,
I suggest
`### CLI Reference` as the second step .

```
### CLI Reference 

Refer to [Command-line interface](https://loopback.io/doc/en/lb4/Command-line-interface.html) for the full list of LoopBack 4 commands.
```

I suggest
`### Adding Additional Dependencies` as the third step .

```
### Adding Additional Dependencies 

Executing CLI commands will automatically update the dependencies in your package.json file, and install them.

If you need to install additional dependencies or dev dependencies, do so with
```

```sh
npm install --save
```

or

```sh
npm install --save-dev
```

The commands above will update the dependencies in your package.json file and install them.

Reply:
Done